### ISSUE TITLE
Test Data

### ISSUE THREAD

**Issue by adamlogan73** [aiolifx/aiolifx#87]
Hey, love the fact that this library exists, and thanks for all of your work on it. I can only imagine the time you guys have put into it.

I saw mention of a 2.0 and I started working on  the 2.0 that I would do, mostly because of my own brain weasels. The thing I am falling short on is I don't see any tests or test data. 

Do you guys happen to have a library of the binary data packets or tests I could run against my version?

These are my changes thus far if you care to check it out: https://github.com/adamlogan73/aiolifx2
Primarily I am moving things to function, and creating dictionaries instead of variables, then moving the message types to Pydantic models (I saw mention of dataclasses, but I fell in love with Pydantic and use it a ton and the backend is written in rust to make it pretty fast)

**Comment by Djelibeybi** [comment]
Yeah, adding tests is one of the things I want to do with 2.0. I have tests for my aiolifx-based libraries, like `aiolifx-themes` but getting tests into this repo is on the list.

**Comment by Djelibeybi** [comment]
I'll take a look at your repo. I currently have both Dataclass-based and Pydantic-based could-be-2.0 branches in my local repo. :) I've also implemented Poetry like you did for package management.

**Comment by adamlogan73** [comment]
Do you have sample packets? I can write tests for v1 so that we can ensure
some backwards compatibility and ensure similar output

**Comment by Djelibeybi** [comment]
Nope, I don't have any tests written yet. All of them are on the TODO list. :)
